code/mod_table/mod_table.cpp

```diff
@@ -31,6 +31,12 @@ bool Damage_impacted_subsystem_first;
 bool Cutscene_camera_displays_hud;
 bool Alternate_chaining_behavior;
 bool Use_host_orientation_for_set_camera_facing;
+bool Use_3d_ship_select;
+bool Use_3d_ship_icons;
+bool Use_3d_weapon_select;
+bool Use_3d_weapon_icons;
+bool Use_3d_overhead_ship;
+int Default_overhead_ship_style;
```

Comment:
Style should really be either an enum, or if it's only ever going to be one of two styles possibly a boolean.

Reply:
It only has two styles right now. If someone adds one in the future, I thought leaving as int would make that simpler.

Reply:
Alright, I have set it up as an enum and added some defensive error checking.